jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -135,19 +138,15 @@ public void setExpandClasspathDependencies(boolean expand) {
   }
 
   @Input
-  @Nullable
```

Comment:
Because `mainClass` was previously nullable, I wonder if we should keep a `Nullable`-annotated getter for it out of consideration for backward compatibility? (Perhaps something similar to when `from.auth.username` and `from.auth.password` were converted for lazy eval)  

https://github.com/GoogleContainerTools/jib/blob/fa8cc958f4ddad55a9eba043fdfd7c32eddb49b6/jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/AuthParameters.java#L50-L52